lib/fork.js

```diff
@@ -59,7 +61,54 @@ class SharedWorkerChannel extends Emittery {
 
 let forkCounter = 0;
 
+const createWorker = (options, execArgv) => {
+	let worker;
+	let port;
+	let _send;
+	let _exit;
+	if (options.workerThreads) {
+		const {port1, port2} = new MessageChannel();
```

Comment:
Why the explicit channel? Can't we use the default channel?

Reply:
Yes we can use the default channel. I didn't because I think it would help to use a custom channel if/when using a worker thread pool.

Example: https://github.com/piscinajs/piscina/blob/current/examples/message_port/index.js

Reply:
That should be a later concern I think.